install/configuring_inventory_file.adoc

```diff
@@ -1104,6 +1104,25 @@ openshift_hosted_routers=[{'name': 'router1', 'certificate': {'certfile': '/path
 
 include::install/topics/glusterfs_intro.adoc[]
 
+You configure {gluster} clusters using variables, which interact with the {product-title} clusters.
+The variables reside in the `[OSEv3:vars]` group and include host variables, role
```

Comment:
> ! reside v
Use “reside” to refer to someone’s place of residence, not as a location for inanimate objects.

I'd say "You define the variables, which include ...,  in the `[OSEv3:vars]` group.

Reply:
Changed to something similar. Thanks.